lib/taskrunner.js

```diff
@@ -5,8 +5,8 @@ const AWS = require('aws-sdk');
 module.exports = {
   makeCmd: function(options) {
     return [
-      'bash', '-c',
-      `bash -c "${options.cmd}"; EXITCODE=$?; echo "TASK FINISHED: $(echo -n ${options.endOfStreamIdentifier} | base64), EXITCODE: $EXITCODE"`
+      'sh', '-c',
```

Comment:
why is this changing?

Reply:
Docker images based on alpine don't ship with bash, sh works fine for this usecase.

Reply:
That's fine. Can you make a separate PR for that change, it's best to keep one PR focussed on a single change in case anything breaks. It's far easier to diagnose any problems.

Reply:
fair enough..